src/main/java/com/conveyal/gtfs/loader/StringListField.java

```diff
@@ -34,7 +35,10 @@ public ValidateFieldResult<String> validateAndConvert(String original) {
     public Set<NewGTFSError> setParameter(PreparedStatement preparedStatement, int oneBasedIndex, String string) {
         // FIXME
         try {
-            Array array = preparedStatement.getConnection().createArrayOf("text", string.split(","));
+            String[] stringList = string.split("(?<=\"),");
+            // Clean the string list of any escaped quotations which are required to preserve any internal commas
```

Comment:
Nit. Full stop.

Reply:
Updated in https://github.com/conveyal/gtfs-lib/pull/386/commits/bc1e1b7051acc7a8c9c149c2453bdf38bb94a2f9